ipynb/tests/Frequency_Invariance_Test.ipynb

```diff
@@ -0,0 +1,799 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Frequency Invariant Load Tracking Test\n",
+    "\n",
+    "FreqInvarianceTest is a LisaTest class for automated testing of frequency invariant load tracking. This notebook uses the methods it provides to perform the same analysis as the automated test and plot some results.\n",
+    "\n",
+    "The test class runs the same workload at a selection of frequencies, each entry in `t.experiments` represents a run at a different frequency."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Setup"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "%matplotlib inline"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "import pandas as pd\n",
+    "import json\n",
+    "\n",
+    "from tests.eas.load_tracking import FreqInvarianceTest\n",
```

Comment:
Maybe this is better to be moved in cell 4 where you actually create the FreqInvarianceTest?
This should make more clear where the test class comes from.